tests/sentry/sentry_metrics/querying/data_v2/test_api.py

```diff
@@ -70,6 +77,8 @@ def setUp(self):
         self.prod_env = self.create_environment(name="prod", project=self.project)
         self.dev_env = self.create_environment(name="dev", project=self.project)
 
+        self.query_transformer = MetricsAPIQueryTransformer()
```

Comment:
For now, tests will use the `MetricsAPIQueryTransformer`, but I do not exclude migrating tests over to directly use the results from the API itself.